README.md

```diff
@@ -122,6 +122,51 @@ fastify.ready(err => {
   fastify.swagger()
 })
 ```
+
+<a name="openapi.spec"></a>
+## OpenAPI spec (specification)
```

Comment:
I think this is not correct. It should be named "OpenAPI validation formats".

Reply:
okey